vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,272 @@
+#Version 0.3
+
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gmps: <http://linked.data.gov.au/def/qld-resource-permit-status/> .
+
+<http://linked.data.gov.au/def/qld-resource-permit-status> a owl:Ontology .
+
+gmps:ConceptScheme a skos:ConceptScheme ;
+    dct:created "2019-09-13"^^xsd:date ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:source "https://myminesonline.business.qld.gov.au" ;
+    skos:definition "The stage in the life cycle of a Queensland Resource Permit."@en ;
+    skos:hasTopConcept gmps:application,
+        gmps:discarded,
+        gmps:granted,
+        gmps:non-current,
+        gmps:non-current-unfinalised,
+        gmps:not-accepted,
+        gmps:prelodgement,
+        gmps:submitted ;
+    skos:prefLabel "Queensland Resource Permit Status"@en .
+
+gmps:abandoned a skos:Concept ;
+    skos:definition "The application has been abandoned."@en ;
```

Comment:
This vocabulary is about resource permits. But the definition has no reference to permits. 
I could assume that this is about the application to be granted a resource permit has being abandoned, or i could assume there was a typo and the permit itself has been abandoned. I think the definitions should be more explicit to distinguish between the grant-of-tenure phases that relate to applications and the actual stages of holding, operating and relinquishing that relate directly to a tenure.

Reply:
According to Jodie: ‘The application has been abandoned’ statement refers to a resource authority application that was not granted, but abandoned by the applicant.